lib/ui/painting/image_dispose_unittests.cc

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
stray newline

Reply:
Fixed.

mac flaked out for reasons not related to this test (Xcode install was bad for some reason and then goma failed). Hopefully this gets a clean run for it.